maqs-selenium/src/test/java/com/cognizantsoftvision/maqs/selenium/SeleniumUtilitiesUnitTest.java

```diff
@@ -24,11 +17,17 @@
 import org.openqa.selenium.support.events.WebDriverListener;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import java.io.File;
```

Comment:
Why we not placing these in alphabetical order?

Reply:
alphabetized the imports